build/Dockerfile.fips

```diff
@@ -0,0 +1,52 @@
+# This version of Go is a Go+BoringCrypto release for FIPS 140-2 compliance
+FROM us-docker.pkg.dev/google.com/api-project-999119582588/go-boringcrypto/golang:1.18.5b7 as build-fips
+WORKDIR /src
+COPY go.mod go.sum /src/
+RUN go mod download -x
+COPY . .
+RUN rm -rf /src/webconsole/dist
+RUN GOOS=linux GOARCH=amd64 WEBCONSOLE=default make immuadmin-static immudb-fips
+RUN mkdir /empty
+
+### distroless FIPS 140-2
+FROM gcr.io/distroless/base:nonroot AS distroless-fips
+LABEL org.opencontainers.image.authors="Codenotary Inc. <info@codenotary.com>"
+
+ARG IMMU_UID="65532"
```

Comment:
Other dockerfiles use 3322 UID/GID, is it safer to use larger values ? I could see a potential issue here if someone would like to easily switch between fips and non-fips image. You would have to assume different UID/GID for both images.

Reply:
actually the nonroot user is created by default for the distroless image with uid/gid 65532, so that is the reason I've used it. Any potential issue?

Reply:
This just uses the default nonroot user created on a distroless image with uid/gid 65532)

Reply:
I just found that there's the user called `nonroot in distroless` - in such case we should avoid using raw UID/GID values (like remove the ARG completely) and use the `nonroot` user name instead. the UID/GID is only necessary where there's really no named user with such UID/GID

Reply:
ack